docs/source/dataframe.rst

```diff
@@ -8,198 +8,181 @@ Dask DataFrame
    :maxdepth: 1
    :hidden:
 
-   dataframe-create.rst
+   Load and Save Data <dataframe-create.rst>
+   Internal Design <dataframe-design.rst>
    Best Practices <dataframe-best-practices.rst>
-   dataframe-design.rst
-   dataframe-groupby.rst
-   dataframe-joins.rst
-   dataframe-indexing.rst
-   dataframe-categoricals.rst
-   dataframe-extend.rst
-   dataframe-parquet.rst
-   dataframe-hive.rst
-   dataframe-sql.rst
-   dataframe-api.rst
-
-A Dask DataFrame is a large parallel DataFrame composed of many smaller pandas
-DataFrames, split along the index.  These pandas DataFrames may live on disk
-for larger-than-memory computing on a single machine, or on many different
-machines in a cluster.  One Dask DataFrame operation triggers many operations
-on the constituent pandas DataFrames.
+   API <dataframe-api.rst>
+   dataframe-extra.rst
 
-.. raw:: html
+.. grid:: 1 1 2 2
 
-   <iframe width="560"
-           height="315"
-           src="https://www.youtube.com/embed/AT2XtFehFSQ"
-           style="margin: 0 auto 20px auto; display: block;"
-           frameborder="0"
-           allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
-           allowfullscreen></iframe>
+   .. grid-item::
+      :columns: 12 12 8 8
 
-Examples
---------
+      Dask DataFrame helps you process large tabular data by parallelizing pandas,
+      either on your laptop for larger-than-memory computing, or on a distributed
+      cluster of computers.
 
-Visit https://examples.dask.org/dataframe.html to see and run examples using
-Dask DataFrame.
+      -  **Just pandas:** Dask DataFrames are a collection of many pandas DataFrames.
 
+         The API is the same.  The execution is the same.
+      -  **Large scale:** Works on 100 GiB on a laptop, or 100 TiB on a cluster.
+      -  **Easy to use:** Pure Python, easy to set up and debug.
 
-.. image:: images/dask-dataframe.svg
-   :alt: Column of four squares collectively labeled as a Dask DataFrame with a single constituent square labeled as a pandas DataFrame.
-   :width: 45%
-   :align: right
+   .. grid-item::
+      :columns: 12 12 4 4
 
-Design
-------
+      .. image:: images/dask-dataframe.svg
+         :alt: Column of four squares collectively labeled as a Dask DataFrame with a single constituent square labeled as a pandas DataFrame.
 
 Dask DataFrames coordinate many pandas DataFrames/Series arranged along the
```

Comment:
The spacing here is a bit off on a big screen (lots of white space above)

![Screenshot 2023-12-15 at 1 21 56 PM](https://github.com/dask/dask/assets/11656932/668bef0a-f83a-4607-ae7e-533811470be5)

I tried moving this paragraph up into the "Just pandas", etc. `grid-item`, which looked better on a big screen, but worse on a small 13" screen. Let's go with that's currently here for now. Just wanted to flag for others, and future me, in case we can do better.